### PR TITLE
NoEmptyPhpdocFixer/PhpdocAddMissingParamAnnotationFixer - missing priority test

### PR DESCRIPTION
**Pull request by SpacePossum**
Add missing priority test "no_empty_phpdoc,phpdoc_add_missing_param_annotation.test".

### CODE REVIEW

File: tests/Fixtures/Integration/priority/no_empty_phpdoc,phpdoc_add_missing_param_annotation.test [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--TEST--
+Integration of fixers: no_empty_phpdoc, phpdoc_add_missing_param_annotation
+--RULESET--
+{"no_empty_phpdoc": true, "phpdoc_add_missing_param_annotation": true}
+--EXPECT--
+<?php
+
```
Comment (keradus) [inline tests/Fixtures/Integration/priority/no_empty_phpdoc,phpdoc_add_missing_param_annotation.test]:
```php
/**
 *
 * @param mixed $a
 */
```

shall be here

**Comment by keradus** [comment]
https://github.com/SpacePossum/PHP-CS-Fixer/pull/22/files

**Comment by SpacePossum** [comment]
awesome, thanks for all the work :)  please squash to one commit with you as author only when merging

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**